sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/BatchManagementClient.cs

```diff
@@ -368,7 +368,7 @@ private void Initialize()
             PrivateEndpointConnection = new PrivateEndpointConnectionOperations(this);
             Pool = new PoolOperations(this);
             BaseUri = new System.Uri("https://management.azure.com");
-            ApiVersion = "2020-03-01";
+            ApiVersion = "2020-05-01";
```

Comment:
Please also update `/sdk/batch/Microsoft.Azure.Management.Batch/AzSdk.RP.props` when you generate from a new API version

Reply:
Updated this and the assembly version in `AssemblyInfo.cs`.